examples/image-folder-support/app.py

```diff
@@ -0,0 +1,215 @@
+import logging
```

Comment:
we can remove these unintended changes if possible to avoid code conflicts.
sorry, this entire folder is probably picked up from my previous change.
i'll need to polish it a little bit before sending a PR out.

Reply:
Good catch